accessibility-swarms.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: page
+title: Accessibility Swarms
+---
+Accessibility Swarms are a practice that we carry out in BBC News.
+
+Swarms take place on **all new front-end code**, this means any changes to HTML, CSS or JavaScript code.
+
+We do this because **accessibility is a team responsibility**.
+
+## When should you carry out a swarm?
+- Before a ticket moves from Development to Test
+- When HTML, CSS & JS is code complete
+- Some developer testing with some assistive technology should already have taken place
+
+## How many people do you need?
+### If it’s a new component or feature
+Carry out a **team swarm**, with 4 to 6 people.
+
+### If it’s a small change
+Carry out a **developer swarm**.
+
+- You don’t need 4 to 6 people
+- The developers pairing on the ticket can work through the **[checklist](accessibility-news-and-testers)** alone
+- You only need to check what’s relevant. For example, if you have only changed a colour, then you only need to check what’s relevant to this. Such as, colour contrast and colour blindness.
+
+## Planning for swarms
+The level of Accessibility Swarm (team or developer swarm) should be defined up front. So that planning sessions can include Accessibility Swarm effort. 
+
+## Who can take part
+- You don’t need to have worked on the ticket
+- All disciplines can join in. The **[testing steps](accessibility-and-testing-with-assistive-technology)** enable anyone, in any role, to use any of the assistive technology for the first time.
+- Ideally you want a mix of roles such as Business Analyst, Designer and Developer
+
+## How to carry out a swarm
+- Get the equipment you need:
+    - Mac laptop
+    - PC laptop with **[supported assistive technology](accessibility-and-supported-assistive-technology)** installed
+    - Headphones with microphone to use with Dragon NaturallySpeaking e.g. Jabra headset
+    - iPhone or iPad
+    - Android phone with Talkback 
+- Grab a space
+- Get into pairs, a technical role with a non-technical role. Such as Developer and Designer working together.
+- Brief the swarmers – Is there something that you need to look out for that might not be obvious? Such as, off screen visually hidden text etc.
+- Work through the **[checklist](accessibility-news-and-testers)**
+- Test with all **[supported assistive technology](accessibility-and-supported-assistive-technology)** following the **[testing steps](accessibility-and-testing-with-assistive-technology)**
+- Make notes of any issues
+- Prioritise bugs
```

Comment:
Could you please link to somewhere that explains prioritisation, or add a section in this file that explains P1/P2/P3 prioritisation.
 In my experience, this is the main area of confusion when it comes to doing a swarm for the first time.

Reply:
It'd be good to add an additional step in this list - 
```- write a ticket in your team's workspace that includes information about replicating the issue & includes the prioritisation level.```

Reply:
I know I need to write something about bug prioritisation, and will do at some point (but not in this PR).